jobs/golangapiserver/spec

```diff
@@ -167,7 +167,7 @@ properties:
     default: 1
   autoscaler.apiserver.scaling_rules.disk.upper_threshold:
     description: "Allowable upper threshold of the disk scaling range"
-    default: 8000
+    default: 10240 # =1024MB*10, 10GB this is the maximum disk quota of CF: https://help.sap.com/docs/btp/sap-business-technology-platform/cloud-foundry-environment
```

Comment:
It could be even better, if we can find a reference of maximum disk in the open source docs :)

Reply:
Good hint! 

I aligned this to be the same default as the default max. app disk size maintained in the Cloud Controller API release, see [d14b3b0](https://github.com/cloudfoundry/app-autoscaler-release/pull/2811/commits/d14b3b0619c57c13b42cc7429dccc0d897640b00). 

In case a deployment of the Cloud Controller API release has overridden the default value of 2048MB, like SAP does it ([there it is 10*1024MB a.k.a. 10G](https://help.sap.com/docs/btp/sap-business-technology-platform/cloud-foundry-environment)), the `autoscaler.apiserver.scaling_rules.disk.upper_threshold` must be set accordingly.